waku/v2/protocol/waku_filter/waku_filter.nim

```diff
@@ -111,13 +114,15 @@ proc init*(T: type FilterRequest, buffer: seq[byte]): ProtoResult[T] =
   for buf in buffs:
     rpc.contentFilters.add(? ContentFilter.init(buf))
 
-  ok(rpc)
+  return ok(rpc)
 
 proc encode*(push: MessagePush): ProtoBuffer =
-  result = initProtoBuffer()
+  var output: ProtoBuffer  = initProtoBuffer()

```

Comment:
No strong feelings about this, but I think the briefer
```suggestion
  var output  = initProtoBuffer()

```
is more idiomatic

Reply:
I know! but personally, I like the explicit type declaration, which makes debugging much easier, and overall enhances the code readability. :) 
I will follow the idiomatic way though!